src/state/store.ts

```diff
@@ -62,12 +77,18 @@ export const createStore = () =>
           ignoredActions: [PERSIST],
         },
       }).concat(apiMiddleware),
-    devTools: true,
+    devTools: {
+      actionSanitizer,
+      stateSanitizer,
+    },
   })
 
 export const store = createStore()
 export const persistor = persistStore(store)
 
+getStateWith(store.getState)
```

Comment:
this was missing causing reselect devtools to be broken

Reply:
maybe there should be a separate PR with just this code that fixes the dev tools rather than combining it with riskier code